src/Core/Models/Config/BotConfiguration.cs

```diff
@@ -87,7 +87,15 @@ public class BotConfiguration
         [YamlMember(Alias = "blockedWords")]
         [Description("A list of regex for words which should automatically be deleted.")]
         public string[] BlockedWords { get; set; }
-        
+
+        [YamlMember(Alias = "spamSettings")]
+        [Description("Settings for the the spam word filter.")]
+        public SpamFilterConfiguration SpamSettings { get; set; }
```

Comment:
Did you verify that this works with the `[]config` command? If I recall correctly I didn't code nested configurations ...

Reply:
It is verified to work now